js/main.js

```diff
@@ -1,330 +0,0 @@
-// main.js
-console.log('main.js')
-
-// global card counter
-let cardNum = 0;
-
-//global array, keep track of zip codes
-zipArray = [];
-
-// Set const to body and assign id with "bodyid"
-let htmlBody = document.querySelector('body');
-htmlBody.setAttribute('id','bodyid');
-
-// Create const mainDiv holds all html elements for weather app
-const mainDiv = document.createElement('div');
-mainDiv.setAttribute('id','main');
-htmlBody.appendChild(mainDiv);
-
-// Create a set of headerDiv and bodyDiv for our content
-// Use as reference for when appending childs
-const headerDiv = document.createElement('div');
-headerDiv.setAttribute('id', 'headerDiv');
-htmlBody.appendChild(headerDiv);
-const bodyDiv = document.createElement('div');
-bodyDiv.setAttribute('id', 'bodyDiv');
-htmlBody.appendChild(bodyDiv);
```

Comment:
Would love to see this in an `init()` function instead of relying on the load of the JS file to run them.